core/arch/aarch64/codec.c

```diff
@@ -940,6 +940,33 @@ encode_opnd_dq5(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *enc_out)
     return encode_opnd_dq_plus(0, 5, 30, opnd, enc_out);
 }
 
+/*
```

Comment:
Wrong comment style here! (Is there a script that checks the way comments are formatted? Of course, the licence notices do not follow the prescribed style.)

Is it safe to approve the commit anyway? I don't think it'll get pushed automatically...

Reply:
Thanks that should be fixed now. Apparently there is no style check rule for comments. 

> Is it safe to approve the commit anyway? I don't think it'll get pushed automatically...
Yep it is safe. Submitting a PR is a separate step, that is done after approval. I will do so in a bit.